docs/source/llm-eval/llm-eval.rst

```diff
@@ -0,0 +1,470 @@
+.. _llm-eval:
+
+====================================
```

Comment:
Can we stick to the standard rst mapping?

<h1> -> =====
<h2> -> ------
<h3> -> ^^^^^
<h4> -> """"""

And we really shouldn't go beyong h4 (even though up to h6 is supported, it looks silly)